tests/StatsdClient.Tests/StatsRouterTests.cs

```diff
@@ -0,0 +1,116 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using StatsdClient;
+using StatsdClient.Aggregator;
+using StatsdClient.Bufferize;
+using StatsdClient.Statistic;
+using StatsdClient.Utils;
+using Tests.Utils;
+
+namespace Tests
+{
+    [TestFixture]
+    public class StatsRouterTests
+    {
+        [Test]
+        public void Routing()
```

Comment:
This function tests 4 different cases (With and without client side aggregation and with and without timestamp). Can you create 4 tests instead? 
Note: You can use `[SetUp]` and `[TearDown]` if you have common setup for your tests.